src/shogun/labels/DenseLabels.h

```diff
@@ -256,7 +262,15 @@ class CDenseLabels : public CLabels
 		 *
 		 * @return label type (binary, multiclass, ...)
 		 */
-		virtual ELabelType get_label_type() const=0;
+		virtual ELabelType get_label_type() const override
+		{
+			return LT_DENSE_GENERIC;
+		}
+
+		virtual const char* get_name() const override
+		{
+			return "DenseLabels";
```

Comment:
@lisitsyn made the class non-abstract, for the sake of loading from files (one doenst know what type of labels it is at that point, only when the algorithm tries to convert to say binary labels we know)